apps/patient/src/views/Pharmacy.tsx

```diff
@@ -522,13 +520,13 @@ export const Pharmacy = () => {
               type = 'COURIER';
               selectedPharmacy = { id: selectedId, name: 'Alto Pharmacy' };
             } else if (selectedId === process.env.REACT_APP_AMAZON_PHARMACY_ID) {
-              type = FulfillmentType.MailOrder;
+              type = 'MAIL_ORDER';
               selectedPharmacy = { id: selectedId, name: 'Amazon Pharmacy' };
             } else if (selectedId === process.env.REACT_APP_COSTCO_PHARMACY_ID) {
-              type = FulfillmentType.MailOrder;
+              type = 'MAIL_ORDER';
               selectedPharmacy = { id: selectedId, name: 'Costco Pharmacy' };
             } else {
-              type = FulfillmentType.PickUp;
+              type = 'PICK_UP';
```

Comment:
why the switch?

Reply:
um, I like strings. But there was a reason at some point, I forget. Will do a second pass later for consistency